<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>takeFiles</class>
 <widget class="QWidget" name="takeFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выберите папку с файлами PRISMA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 252, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1-кластер</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Введите папку c нужными файлами</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>35</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>2-кластер</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Введите папку c нужными файлами</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>105</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Обзор</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
